pkg/registry/apis/secret/service/secret.go

```diff
@@ -43,6 +45,7 @@ func (s *SecretService) Create(ctx context.Context, sv *secretv0alpha1.SecureVal
 		out = createdSecureValue
 
 		if _, err := s.outboxQueue.Append(ctx, contracts.AppendOutboxMessage{
+			RequestID: contracts.GetRequestId(ctx),
```

Comment:
The only thing I am confused about is where the `requestId` is coming from? If when appending to the outbox queue we get it from the context, and then on the worker we populate the context with the `message.RequestID`, what generates it?